app/src/main/res/values/strings.xml

```diff
@@ -1874,6 +1874,8 @@
     <string name="preferences_app_protection__change">Change</string>
     <string name="preferences_app_protection__create">Create</string>
     <string name="preferences_app_protection__your_pin_adds_an_extra_layer_of_security_and_backs">Your PIN adds an extra layer of security and backs up your account. You\'ll be asked for it when you register your phone number with Signal. If you forget your PIN, you\'ll be locked out of your account for 7 days.</string>
+    <string name="preferences_app_protection__disable_pinv2_reminders">Disable PIN reminders</string>
+    <string name="preferences_app_protection__disable_pinv2_reminders_summary">Disable PIN reminder requests</string>    
```

Comment:
The summary could be more explicit like "Do not prompt to enter your PIN periodically"

Reply:
OK.